llvm/lib/SandboxIR/SandboxIR.cpp

```diff
@@ -180,6 +206,19 @@ bool User::classof(const Value *From) {
   }
 }
 
+void User::setOperand(unsigned OperandIdx, Value *Operand) {
+  if (!isa<llvm::User>(Val))
```

Comment:
`assert(isa<llvm::User>(Val) && "No operands!");`

Reply:
and that's sorta already covered by the `cast<llvm::User>` below, but I guess a nicer error message is good

Reply:
Yes, let's keep it.